frontend/packages/console-shared/src/components/topology/TopologySideBarTabSection.tsx

```diff
@@ -0,0 +1,8 @@
+import * as React from 'react';
+import './SideBarTabSection.scss';
+
+const SideBarTabSection: React.FC = ({ children }) => {
```

Comment:
Why not let extension contributor contribute the children part and we control the `SideBarTabSection` from topology?